packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -95,8 +100,28 @@ describe("Compile task", () => {
   it("should recompile all files when --force is used", async () => {
     await op.resetAndCompile(true);
 
-    assert.equal(op.timestamp, 5);
-    assert.lengthOf(op.compiledFiles, 2);
-    assert.lengthOf(op.writtenFiles, 2);
+    assert.equal(op.timestamp, 9);
+    assert.lengthOf(op.compiledFiles, 4);
+    assert.lengthOf(op.writtenFiles, 4);
+  });
+
+  it("should return TEAL code", async () => {
+    const pyOp = new PyCompileOp(op);
+    const content = fs.readFileSync(path.join(ASSETS_DIR, f4), 'utf8');
+    const res = pyOp.compilePyTeal(f3PY) + '\n';
+    assert.deepEqual(content.toString(), res);
+  });
+
+  it("should return correct PyASCCache from CompileOp", async () => {
+    const result = await op.ensureCompiled(f3PY, true);
+    const expected = fs.readFileSync(path.join(ASSETS_DIR, 'gold-asa-py-check.yaml'), 'utf8');
+    assert.deepEqual(YAML.stringify(result), expected);
+  });
+
+  it("should return correct PyASCCache from PyCompileOp", async () => {
+    const pyOp = new PyCompileOp(op);
+    const result = await pyOp.ensureCompiled(f3PY, false);
+    const expected = fs.readFileSync(path.join(ASSETS_DIR, 'gold-asa-py-check.yaml'), 'utf8');
```

Comment:
we should remove the file before starting the test. Otherwise we don't know if the file was there because of the previous test, or this test.

Reply:
this file is for comparing the result, we will get the result in previous line, this is expected file